### PR TITLE
Infinex Core Blast deployment

### PR DESCRIPTION
**Pull request by lemoninfinex**

### CODE REVIEW

File: infinex-multichain/mainnets/infinex-multichain-blast.toml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+name = "infinex-core"
+version = "3.0.0"
+preset = "core"
+description = "Infinex Core Deployments"
+#  pnpm cannon build infinex-multichain/mainnets/infinex-multichain-ethereum.toml --chain-id 81457 --dry-run --upgrade-from
+#  pnpm cannon publish infinex-core-test:3.0.0@core --chain-id 1 --exclude-cloned
```
Comment (rayInfinex) [inline infinex-multichain/mainnets/infinex-multichain-blast.toml:6]: pls update this to use infinex-core and the right chain id

**Comment by zeus-infinex** [comment]
## [testnets](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/aff5f42/report/testnets/) <span title="passed">✅173</span> | <span title="failed">❌2</span>
[Markdown](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/aff5f42/report/testnets/report.md) | [JSON](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/aff5f42/report/testnets/report.json)
<details>
<summary>craterun</summary>

### Craterun (testnets) Opener recognises keeper from Platform app
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/craterun.test.ts#L108

```
InvalidAddressError: Address "undefined" is invalid.
```

### Craterun (testnets) CTP staking rewards recognises keeper
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/craterun.test.ts#L116

```
InvalidAddressError: Address "undefined" is invalid.
```
</details>


## [staging](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/aff5f42/report/staging/) <span title="passed">✅178</span> | <span title="failed">❌2</span>
[Markdown](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/aff5f42/report/staging/report.md) | [JSON](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/aff5f42/report/staging/report.json)
<details>
<summary>craterun</summary>

### Craterun (staging) Opener recognises keeper from Platform app
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/craterun.test.ts#L108

```
InvalidAddressError: Address "undefined" is invalid.
```

### Craterun (staging) CTP staking rewards recognises keeper
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/craterun.test.ts#L116

```
InvalidAddressError: Address "undefined" is invalid.
```
</details>


## [mainnets](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/aff5f42/report/mainnets/) <span title="passed">✅178</span> | <span title="failed">❌2</span>
[Markdown](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/aff5f42/report/mainnets/report.md) | [JSON](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/aff5f42/report/mainnets/report.json)
<details>
<summary>craterun</summary>

### Craterun (mainnets) Opener recognises keeper from Platform app
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/craterun.test.ts#L108

```
InvalidAddressError: Address "undefined" is invalid.
```

### Craterun (mainnets) CTP staking rewards recognises keeper
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/craterun.test.ts#L116

```
InvalidAddressError: Address "undefined" is invalid.
```
</details>

File: infinex-multichain/mainnets/infinex-multichain-blast.toml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+#  pnpm cannon verify infinex-core-test:3.0.0@core --chain-id 1 --api-key
+
+[var.Deploy]
+DEPLOYER = "0x6298551D56F825B3b6c3350D32129Ffd3cFb198C"
```
Comment (rayInfinex) [inline infinex-multichain/mainnets/infinex-multichain-blast.toml:10]: deployer needs to be changed to the actual deployer address

File: infinex-multichain/mainnets/infinex-multichain-blast.toml (hunk 1 above)
Comment (lemoninfinex) [reply infinex-multichain/mainnets/infinex-multichain-blast.toml]: done

**Review by ramboinfinex — APPROVED** [review]

**Review by rayInfinex — APPROVED** [review]

**Pull request merged**